cider/exec/module/batch/CiderBatchUtils.cpp

```diff
@@ -19,13 +19,14 @@
  * under the License.
  */
 
+#include <memory>
```

Comment:
remove? I think `CIDERBATCH_WITH_ARROW` should always be first.

Reply:
fix， the same before